packages/dcos-integration-test/extra/test_endpoints.py

```diff
@@ -141,3 +141,52 @@ def test_if_we_have_capabilities(cluster):
     )
     assert r.status_code == 200
     assert {'name': 'PACKAGE_MANAGEMENT'} in r.json()['capabilities']
+
+
+def test_if_overlay_master_is_up(cluster):
+    r = cluster.get('/mesos/overlay-master/state')
+    assert r.ok, "status_code: {}, content: {}".format(r.status_code, r.content)
+
+    # Make sure the `dcos` overlay has been configured.
+    json = r.json()
+
+    dcos_overlay_network = {
+        'vtep_subnet': '44.128.0.0/20',
+        'vtep_mac_oui': '70:B3:D5:00:00:00',
+        'overlays': [
+            {
+                'name': 'dcos',
+                'subnet': '9.0.0.0/8',
+                'prefix': 24
+            }
+        ]
+    }
+
+    assert 'network' in json
+    assert 'overlays' in json['network']
+    assert json['network'] == dcos_overlay_network
+
+
+def test_if_overlay_master_agent_is_up(cluster):
+    r = cluster.get('/mesos/overlay-agent/overlay')
+    assert r.ok, "status_code: {}, content: {}".format(r.status_code, r.content)
+
+    # Make sure the `dcos` overlay has been configured.
+    json = r.json()
+    dcos_overlay = {
+        'name': 'dcos',
+        'subnet': '9.0.0.0/8',
+        'prefix': 24
+    }
+
+    assert 'overlays' in json
```

Comment:
Wasn't aware that you could do dictionary comparisons by excluding keys.
Atleast without iterating through the keys. Is that what you were
suggesting?